docker/zk-environment/20.04_amd64_cuda_11_8.Dockerfile

```diff
@@ -0,0 +1,183 @@
+FROM ubuntu:20.04@sha256:3246518d9735254519e1b2ff35f95686e4a5011c90c85344c1f38df7bae9dd37 as base
+
+WORKDIR /usr/src/zksync
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install required dependencies
+RUN apt-get update && apt-get install -y \
+    cmake \
+    make \
+    bash \
+    git \
+    openssl \
+    libssl-dev \
+    gcc \
+    g++ \
+    curl \
+    pkg-config \
+    software-properties-common \
+    jq \
+    openssh-server \
+    openssh-client \
+    wget \
+    vim \
+    ca-certificates \
+    gnupg2 \
+    postgresql-client \
+    wget \
+    bzip2
+
+# Install dependencies for RocksDB. `liburing` is not available for Ubuntu 20.04,
+# so we use a PPA with the backport
+RUN add-apt-repository ppa:savoury1/virtualisation && \
+    apt-get update && \
+    apt-get install -y \
+    curl \
```

Comment:
Do we need both wget and curl?

Reply:
consider these as:
1. dev/debug tools
2. weren't added just now, the actual diff here is "file got moved/renamed"